api/app/models/phone_number.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+class PhoneNumber < ApplicationRecord
```

Comment:
Add [Associations](http://guides.rubyonrails.org/association_basics.html) to the PhoneNumber and User Models.

Reply:
To be able to say User.phone_numbers.